testing/file_test/file_test_base.h

```diff
@@ -30,7 +30,7 @@ namespace Carbon::Testing {
 // ...` and `// CHECK:STDERR: ...` respectively. `[[@LINE+offset]` and
 // `{{regex}}` syntaxes should also work.
 //
-// `lit_autoupdate.py` automatically constructs compatible lines.
+// `autoupdate_lit_testdata.py` automatically constructs compatible lines.
```

Comment:
```suggestion
// `autoupdate_testdata.py` automatically constructs compatible lines.
```

The one with "lit" in the name is for `lit_testdata` (explorer only), rather than `testdata` which this file is for.